src/playground/dist/public/js/app.js

```diff
@@ -0,0 +1,66 @@
+function registerAskScriptEditor(
```

Comment:
Would it be possible to do this in TypeScript?

Reply:
Also, there seems to be a copy of this file in `src/playground/src/public/assets/js/app.js`